nixos.nix

```
@@ -117,22 +120,26 @@ in
             sourceBase="''${sourceBase%/}"
             target="''${target%/}"
 
+            (
+            # check that the source exists
+            realSource="$(realpath "$sourceBase$target")"
+            if [ ! -d "$realSource" ]; then
+                printf "\e[1;31mBind source '%s' does not exist!\e[0m\n" "$realSource"
+            fi
+            )
+            (
```

Would improve readability to indent all the stuff inside each scope to match its nesting level.

Also:
```suggestion
            )
            
            (
```